nifi-docker/dockerhub/Dockerfile

```diff
@@ -44,7 +44,9 @@ RUN groupadd -g ${GID} nifi || groupmod -n nifi `getent group ${GID} | cut -d: -
     && mkdir -p ${NIFI_BASE_DIR} \
     && chown -R nifi:nifi ${NIFI_BASE_DIR} \
     && apt-get update \
-    && apt-get install -y jq xmlstarlet procps
+    && apt-get install -y jq xmlstarlet procps \
+    && apt-get clean  \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
This step removes the apt lists which is an unnecessary step that could cause confusion for people who extend the docker image.